src/applications/disability-benefits/view-payments/components/view-payments-lists/payments-returned/PaymentsReturned.jsx

```diff
@@ -1,44 +1,45 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
I think this Component import can be removed.

Reply:
removed